tests/ballerina-test-integration/src/test/resources/transaction/participant2.bal

```diff
@@ -30,13 +30,9 @@ endpoint http:Listener participant2EP {
 //};
 
 endpoint sql:Client testDB {
-    database: sql:DB_HSQLDB_SERVER,
-    host: "localhost",
-    port: 9001,
-    name: "TEST_SQL_CONNECTOR",
+    url: "hsqldb:hsql://localhost:9001/TEST_SQL_CONNECTOR",
```

Comment:
It is better to have database: sql:DB_HSQLDB_SERVER, rather than exposing JDBC details here.